data/human/human missions.txt

```diff
@@ -3996,3 +3996,285 @@ mission "Fumbling Mugger"
+			`	One is missing; Mussie.`
+			`	You turn the ship back the way you came and pull up the sensor recording to see what happened. The flash of light came from viney seaweed, stretching as far up and down as the ship can detect. Evidently, contact with it released a jolt of electricity that lit up both the vine and Mussie's subpod.`
+				goto rollcall
+			label dead
+			`	He reads the riddle to you. "There are stingers in the deep \ A certain death, a dreamless sleep. \ Go to beast that has no bee \ No danger there, just open sea."`
+			choice
+				`	(Go east.)`
+					goto east2
+				`	(Go north.)`
+					goto north2
+				`	(Go west.)`
+			action
+				set "Mussie dead"
+			`	The proximity alarm wails and this time, you're much quicker to stop the ship... but Mussie isn't so lucky. His subpod smashes straight into a different stretch of the same rock wall that claimed Elias and Taddese not minutes before.`
+			`	Again, you put the ship into reverse and turn around. This is not going at all how it was supposed to.`
+				goto rollcall
+			label east2
+			`	Cautiously, you turn to the east and power forward. Subpods fly ahead, exulting in their agility in the face of the dangerous ocean.`
+			`	After holding your breath for what feels like hours, Karengo radios you. "Knew you'd find your way eventually. What's the next direction?"`
+				goto riddlethree
+			label north2
+			action
+				set "Mussie dead"
+			`	Bubbles rush past the thick windows as the group's subpods scout to the north at your direction. At first, all is quiet. Just as you start to exhale a sigh of relief, twin lightning bolts arc across the waterscape ahead and the ship sensors overload, blinded by an apparent release of electricity. The systems recover, showing subpods returning quickly back to the ship.`
+			`	One is missing; Mussie.`
+			`	You turn the ship back the way you came and pull up the sensor recording to see what happened. The flash of light came from viney seaweed, stretching as far up and down as the ship can detect. Evidently, contact with it released a jolt of electricity that lit up both the vine and Mussie's subpod.`
+			label rollcall
+			branch dead1
+				has "Elias and Taddese dead"
+			`	Karengo sounds a roll call over the radio. Mussie never reports in, and his subpod beacon isn't responding. "Ah, well." says Karengo, without much sympathy. "You all stay here and I'll scout ahead."`
+			`	Only minutes later, he's found a clear path east, and you morosly trail the ship behind.`
+				goto riddlethree
+			label dead1
+			`	Karengo sounds a roll call over the radio. Elias and Taddese are still missing, and now Mussie has failed to report in as well. None of their subpod beacons are responding.`
```

Comment:
For `dead1`, the only condition is Elias and Taddese are dead. Is Mussie dead too?

Reply:
Should be fixed now. Thanks for catching this!